velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h

```diff
@@ -17,8 +17,14 @@
 
 namespace facebook::velox::filesystems {
 struct HdfsServiceEndpoint {
+  HdfsServiceEndpoint(std::string host, std::string port) {
+    this->host = host;
+    this->port = atoi(port.data());
+    this->identity = host + (port.empty() ? "" : ":" + port);
```

Comment:
Thanks for your comment! Yes, we can make it as a simple function, instead of unnecessarily holding an identity member. Just fixed it in the new commit.